src/main/scala/com.socrata.tileserver/util/RenderProvider.scala

```diff
@@ -27,18 +32,21 @@ import RenderProvider._
 case class RenderProvider(http: HttpClient, baseUrl: RequestBuilder) {
   /** Render the provided tile using the provided request info.
     *
-    * @param rawTile a Map that contains the features as WKB.
+    * @param encoder An encoder to manipulate features
     * @param info the request info to use while rendering the tile.
     */
-  def renderPng(rawTile: MapTile, info: RequestInfo): InputStream = {
-    val style = info.style.get
-    val tile: Map[String, Seq[String]] = rawTile.map { case (layer, wkbs) =>
-      layer -> wkbs.map(Base64.encodeBase64String(_))
+  def renderPng(encoder: TileEncoder, info: RequestInfo): InputStream = {
```

Comment:
We had one approach that required having the raw features but we refactored that probably do not need to pass the encoder anymore (yay to less tests to fix).